src/services/ContentService.js

```diff
@@ -565,15 +565,21 @@ define(["structures/ContentCreateStruct", "structures/ContentUpdateStruct", "str
      *
      * @method loadUserDrafts
      * @param userId {String} target user identifier (e.g. "/api/ezp/v2/user/users/14")
+     * @parma [headers] {Object} additional headers
```

Comment:
the `[` and `]` around `headers` means the parameter is not mandatory so that's kind of implied but to be honest, the documentation was not really my main purpose here :-)
That said, this is a good point because we could also do the same thing by keeping 2 parameters on the method and allowing the first one to be an option object:

```js
/**
  * bla bla
  *
  * @param {Object|String} options the userId or an object containing the userId and others options
  * @param {String} options.userId
  * @param {Object} options.headers additional headers
  * @param {Function} callback
  */
ContentService.prototype.loadUserDrafts = function (options, callback) {
    var userId = options, headers = {};

    if ( typeof options === "object" ) {
        userId = options.userId;
        headers = options.headers ? options.headers : {};
    }
    // ...
};
```

and I'm not really sure which approach is the best